screens/SendFeedbackScreen.tsx

```diff
@@ -202,7 +223,12 @@ export default function SendFeedbackScreen({ navigation }) {
       />
       <Text style={styles.sectionTitle}>Image Upload</Text>
       <View style={styles.imageUploadWrapper}>
-        {!selectImage && (
+        {images.map((item, index) => (
+          <TouchableOpacity key={index} onPress={() => editImage(index)}>
+            <Image style={styles.chosenImage} source={{ uri: item }} />
```

Comment:
We can maybe rename `item` to `imageURI` to make this a bit more readable. Also, I don't see any option to delete images, only to upload and edit -- what if someone uploads 3 images but then only wants to submit 2 of them?

Reply:
I'll check with design on that. I wasn't able to find a design that showed image deletion, but that is def a good call